app/LiveEvent.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class LiveEvent extends Model {
```

Comment:
As mentioned in a previous comment, I'd still like to see if a private constructor is possible here

Reply:
Done - added.